app/src/main/java/openfoodfacts/github/scrachx/openfood/network/OpenFoodAPIClient.java

```diff
@@ -304,7 +304,7 @@ public void onResponse(Call<AllergenRestResponse> call, Response<AllergenRestRes
 
             @Override
             public void onFailure(Call<AllergenRestResponse> call, Throwable t) {
-
+                onAllergensCallback.onAllergensResponse(false);
```

Comment:
Indeed it allows the user to reach the MainActivity in case of network error downloading `allergens.json` from the server on start